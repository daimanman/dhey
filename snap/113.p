{"C":1,"Duration":28922744,"EndTime":1551409700213524600,"Err":"","FileId":"","InputFileName":"","LonhApi":"sendTest","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409700184601900,"Status":1,"TaskId":113,"Type":"LONH","Url":"","Z":""}