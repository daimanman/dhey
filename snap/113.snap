{"AvgTotal":0.0179509,"Fastest":0.0179509,"Slowest":0.0179509,"Average":0.0179509,"Rps":55.70751327231504,"AvgConn":0.0010101,"AvgDNS":0,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0169408,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0169408,"DelayMin":0.0169408,"Lats":[0.0179509],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0169408],"Total":17950900,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":74,"SizeReq":74,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0179509,"Count":1,"Frequentcy":1},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0},{"Mark":0.0179509,"Count":0,"Frequentcy":0}],"TaskId":113,"Duration":0.0179509}